vru_use_case_scripts/README.md

```diff
@@ -142,3 +142,37 @@ Plot the data:
 ### Example output
 
 ![](docs/plot_time_to_collision_example.png)
+
+## `calc_post_encroachment-time`
+
+This script takes in two CSV files containing vehicle and pedestrian odometry generated from rosbags. It calculates
```

Comment:
Capture simplifying assumptions (e.i 5x5 or configurable size box from intersection of vehicle and pedestrian)